DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Indigo\Bundle\CookieConsentBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('indigo_cookie_consent');
+
+        $rootNode
+            ->children()
+                ->arrayNode('options')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('message')->defaultValue('message')->end()
+                        ->scalarNode('dismiss')->defaultValue('dismiss')->end()
+                        ->scalarNode('learnMore')->defaultValue('learn_more')->end()
```

Comment:
ah, i should have read the PR description, sorry.

hm, i would say screw them :-) i would put the translation calls into the twig template, not allow any configuration for that and tell users to override the template if they disable translations. keeps things a lot simpler and imho even for mono-language websites you should use translations to separate semantical meaning from actual translation. it helps knowing what text snippets are used on a website.